spec/support/suspenders.rb

```diff
@@ -12,9 +12,10 @@ def create_tmp_directory
   def run_suspenders(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        ENV['TESTING'] = '1'
-
-        %x(#{suspenders_bin} #{APP_NAME} #{arguments})
+        `
```

Comment:
Not really sure, but here's [the comment](https://github.com/thoughtbot/suspenders/pull/646#discussion_r43736087).